.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: "WebGL SDK Publish"
+
+on:
+    push:
```

Comment:
more of an open question: should this be every push or every release?

Using Github tags and releases:
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

We can sync the npm releases to our github release tags without needing a new tag for every push.

Big picture, do we want to release with every push or plan releases independently?

Reply:
I made the required changes to trigger the publishing on release only. That will be more convenient for release planning.